src/Npgsql/NpgsqlConnector.cs

```diff
@@ -1042,14 +1042,22 @@ internal ValueTask<IBackendMessage> ReadMessage(bool async, DataRowLoadingMode d
                 }
 
                 PostgresException? error = null;
+                CancellationTokenSource? cts = null;
```

Comment:
Let's rename to `timeoutCts` and `timeoutCt` to prevent confusion with any user-supplied ct.